es6-classes/square.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(width) { super(); }
```

Comment:
You should also be defining `this.width` with the value that gets passed in from the `width` parameter